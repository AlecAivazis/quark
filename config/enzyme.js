// external imports
import Enzyme from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
import browserEnv from 'browser-env'

// add a request animation frame polyfill
import raf from 'raf'
raf.polyfill()

// enable the react 16 adapter for enzyme
Enzyme.configure({ adapter: new Adapter() })

// add various browser mocks
browserEnv()
