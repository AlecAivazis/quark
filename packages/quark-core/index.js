export * from './components';
import * as _styles from './styles';
export { _styles as styles };