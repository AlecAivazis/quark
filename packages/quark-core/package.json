{
    "name": "quark-core",
    "version": "1.0.6",
    "module": "index.js",
    "main": "index.js",
    "dependencies": {
        "react-native-web": "0.3.4",
        "svgs": "^3.1.1"
    },
    "peerDependencies": {
        "react": "^16.3.0"
    }
}
