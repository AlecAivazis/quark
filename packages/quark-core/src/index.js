// @flow
export * from './components'
export * as styles from './styles'
