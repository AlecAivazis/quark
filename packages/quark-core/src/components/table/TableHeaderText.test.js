// external imports
import React from 'react'
import { Text } from 'react-native'
import { mount } from 'enzyme'
// local imports
import { TableHeaderText } from 'quark-core'

it('renders', () =>
    expect(
        mount(
            <TableHeaderText>
                <Text>hello</Text>
            </TableHeaderText>
        )
    ).toMatchSnapshot())
