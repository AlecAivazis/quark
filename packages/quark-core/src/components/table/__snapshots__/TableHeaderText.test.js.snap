// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<TableHeaderText>
  <GetTheme>
    <Text
      style={
        Object {
          "color": "#A7A7A7",
        }
      }
    >
      <div
        className="rn-borderTopWidth-13yce4e rn-borderRightWidth-fnigne rn-borderBottomWidth-ndvcnb rn-borderLeftWidth-gxnn5r rn-boxSizing-deolkf rn-display-1471scf rn-font-1lw9tu2 rn-fontFamily-10u92zi rn-fontSize-ubezar rn-marginTop-1mnahxq rn-marginRight-61z16t rn-marginBottom-p1pxzi rn-marginLeft-11wrixw rn-paddingTop-wk8lta rn-paddingRight-9aemit rn-paddingBottom-1mdbw0j rn-paddingLeft-gy4na3 rn-textDecoration-bauka4 rn-whiteSpace-q42fyq rn-wordWrap-qvutc0"
        dir="auto"
        style={
          Object {
            "color": "rgba(167,167,167,1)",
          }
        }
      >
        <Text>
          <span
            className="rn-borderTopWidth-13yce4e rn-borderRightWidth-fnigne rn-borderBottomWidth-ndvcnb rn-borderLeftWidth-gxnn5r rn-boxSizing-deolkf rn-color-homxoj rn-display-1471scf rn-font-1lw9tu2 rn-fontFamily-poiln3 rn-fontSize-7cikom rn-marginTop-1mnahxq rn-marginRight-61z16t rn-marginBottom-p1pxzi rn-marginLeft-11wrixw rn-paddingTop-wk8lta rn-paddingRight-9aemit rn-paddingBottom-1mdbw0j rn-paddingLeft-gy4na3 rn-textDecoration-bauka4 rn-whiteSpace-irrty rn-wordWrap-qvutc0"
            dir="auto"
          >
            hello
          </span>
        </Text>
      </div>
    </Text>
  </GetTheme>
</TableHeaderText>
`;
